packages/react-native-editor/package.json

```diff
@@ -31,8 +31,8 @@
 	"dependencies": {
 		"@babel/runtime": "^7.13.10",
 		"@react-native-community/blur": "3.6.0",
-		"@react-native-community/masked-view": "git+https://github.com/wordpress-mobile/react-native-masked-view.git#v0.1.11-wp",
-		"@react-native-community/slider": "git+https://github.com/wordpress-mobile/react-native-slider.git#v3.0.2-wp",
+		"@react-native-community/masked-view": "git+https://github.com/wordpress-mobile/react-native-masked-view.git#wp-fork",
```

Comment:
I think we need to not use `wp-fork` here because even though there is a `wp-fork` tag, it conflicts with the `wp-fork` branch on that repo and in my testing, npm will pull the branch, not the tag, when they conflict. 

[Related comment](https://github.com/wordpress-mobile/react-native-hsv-color-picker/pull/3#issuecomment-902950210)

Reply:
@mchowning @illusaen Shouldn't it be `v0.1.12-wp`, `v0.2.11-wp` or something similar, because the `-wp` ones are our tags, right?

I think @ceyhun might have been the one who worked on the previous tags for react-native `0.64` upgrade, maybe he can advise on how to proceed?

This is one of the reasons I left the tagging/updating the references part to gutenberg-mobile developers, because I thought there was a process in place. And if there isn't one, maybe we can settle on one and document it now?

Reply:
In general, I believe that the `v###-wp` signifies that it is `v###` of the library, and[ we add the `-wp` to signify it is from our fork](https://github.com/wordpress-mobile/react-native-masked-view/commit/959000abf16d7a024c9c084af7a9fc7d86293573). The issue I was thinking we would have with that is that if we didn't update the library itself to a new version (i.e., only added our gradle updates), we couldn't follow that same scheme because the library version is unchanged so it would be the same tag (I guess technically we could "move" the tag, but that's a bad idea for a number of reasons). With that said, as you point out `masked-view` is a new version for some reason (and it wasn't updated through a PR, so we probably need to lock down those `wp-fork` branches to only be updated by PRs), which I wasn't expecting. 

So maybe I am missing something here. I need to look into what is going on here to understand it a bit better. If we are bumping library versions that will probably require some extra testing.

Any chance you may have updated some of the `wp-fork` branches when you were working on updating the dependencies Paul?